### PR TITLE
feature(UploadProfileImage): add endpoint

### PR DESCRIPTION
**Pull request by lucjac**

### CODE REVIEW

**Review by fadani — APPROVED** [review]

File: src/version-5/upload-profile-image.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+export interface PayloadUploadProfileImageRequestV5 {
+    readonly image: string
```
Comment (fadani) [inline src/version-5/upload-profile-image.ts:3]: would this be a base64 encoded image string?

Comment (fadani) [reply src/version-5/upload-profile-image.ts:3]: because then maybe we could add `base64` somehow to the name of this property to make it explicit

Comment (lucjac) [reply src/version-5/upload-profile-image.ts:3]: yeah that was my question to Anton :D i dont know, probably that would be the easiest?

Comment (fadani) [reply src/version-5/upload-profile-image.ts:3]: ah okay, makes sense :D

**Pull request merged**